# libxml2 has a problem when a document encoding is specified, even though
# documentation says that all internal representation would be in UTF-8 and
# language-subtag-registry data already is passed in UTF-8, when specifying the
# document encoding to be UTF-8 the output to language-subtag-registry.xml is
# conversion garbage, '?' on Linux UTF-8 locale, ISO-8859-1 on Windows. If no
# document encoding is specified at all no conversion happens and non-ASCII
# values are correctly written as entities.

--- misc/liblangtag-0.4.0/data/reg2xml.c	2012-01-30 13:20:57.000000000 +0100
+++ misc/build/liblangtag-0.4.0/data/reg2xml.c	2012-08-11 02:30:39.147779753 +0200
@@ -134,7 +134,6 @@
 	lt_bool_t retval;
 
 	doc = xmlNewDoc((const xmlChar *)"1.0");
-	doc->encoding = xmlStrdup((const xmlChar *)"UTF-8");
 	root = xmlNewDocNode(doc, NULL,
 			     (const xmlChar *)"registry",
 			     NULL);
